src/toil/provisioners/contrib/azure_rm.py

```diff
@@ -0,0 +1,859 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2016 Matt Davis, <mdavis@ansible.com>
+#                    Chris Houseknecht, <house@redhat.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Unfortunately, according to some legal interpretations, [we can't include GPL-licensed software as part of an Apache-licensed product](https://www.apache.org/licenses/GPL-compatibility.html) (though the other way around is OK). I'm not a lawyer, but from the little I know, including this source file could potentially force the entire project to be GPL-licensed. (I have no idea what would happen if you downloaded it using wget or whatever.)

Reply:
No, we can not include this file in Toil distribution. Why is in in a contriib diorectory?

Reply:
The original directory included `contrib`: https://github.com/ansible/ansible/blob/devel/contrib/inventory/azure_rm.py

Toil already has a bunch of Azure stuff in a `contrib` folder here too: https://github.com/BD2KGenomics/toil/tree/master/contrib/azure  but I don't know if anyone has used it or if it's just the remnants of a failed implementation at this point.

Reply:
Good point. Thanks! This was going to be removed in the refactoring, but I will address it now.